monitoringlinter/go.mod

```diff
@@ -0,0 +1,7 @@
+module github.com/monitoring/monitoringlinter
+
+go 1.22.0
```

Comment:
please keep it on go1.21 for now. 

It's going to be used in projects that are still using 1.21 and these project won't be able to use this package.

Reply:
Done